src/serialization.cpp

```diff
@@ -152,6 +152,18 @@ size_t JsonWriteRichPresenceObj(char* dest,
                     WriteOptionalString(writer, "join", presence->joinSecret);
                     WriteOptionalString(writer, "spectate", presence->spectateSecret);
                 }
+                else 
+                {
+                    if (presence->buttonUrl[0] && presence->buttonLabel[0] ||
```

Comment:
Please use () when && and || are involved in the same expression.

Reply:
Sure.